tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
Precisely. And without this code we're failing to test that.

What happens instead is that we keep feeding the RFB code new buffers. But it is waiting for the browser, so all it can do is queue up things. So after a short while we're mostly testing the speed of buffer management as the buffers grow to unrealistic proportions.

A real server would either notice the lack of framebuffer update requests, or lack of fence responses, and hold off on sending more updates. So the performance code needs something similar.